jsond/__init__.py

```diff
@@ -39,7 +39,7 @@ def str_to_datetime(self, s):
             return s
 
     def _decode(self, data):
-        if isinstance(data, basestring) and data.startswith('datetime:'):
+        if isinstance(data, str) and data.startswith('datetime:'):
```

Comment:
Ah sorry, I'm doing commits where I do `2to3 -w -n .` on the repo, and then I go through and `six` all the things that change!